### ISSUE TITLE
Unknown rule indentation

### ISSUE THREAD

**Issue by luis-pato** [stylelint/stylelint#8025]
After updating the stylelint and stylelint-config-standard packages i get a couple of errors on all my css files:
- `Unknown rule indentation`
- `Unknown rule number-leading-zero`

Before updating the packages the linter was working without problems.
I will add the information for the 'Unknown rule indentation" problem, but the same applies also for the number-leading-zero problem...

### What minimal example or steps are needed to reproduce the bug?

```css
.my selector {
    border: 1px solid red;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "indentation": 4
  }
}
```

### How did you run Stylelint?

```shell
npx stylelint ./src/styles/**/*.css
```

### Which Stylelint-related dependencies are you using?

```json
{
    "stylelint": "^16.9.0",
    "stylelint-config-standard": "^36.0.1",
}
```

### What did you expect to happen?

I expected the linter to find indentation problems

### What actually happened?

I get the following error for all my css files:

```
   1:1  ✖  Unknown rule indentation                      indentation
```

**Comment by Mouvedia** [comment]
Hi @luis-pato,

`indentation` and `number-leading-zero` [were deprecated in version 15](https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules) and [removed in version 16](https://stylelint.io/migration-guide/to-16#removed-deprecated-stylistic-rules).

**Comment by luis-pato** [comment]
@Mouvedia Thank you for your quick answer. I have missed that.
I will look for an alternative.

I will close the ticket!

**Comment by seahindeniz** [comment]
@Mouvedia is there a plan for adding them back? Because there's no alternative that works, lack of the removed rules are creating more DX problems.

As a specific case, the `prettier` alternative doesn't work on frameworks because of this line https://github.com/prettier/stylelint-prettier/blob/157628a70a0b62fadc3626bc287d035ca59828b5/index.js#L84-L94

I have updated deps as 
```json
        "stylelint": "15.11.0",
        "stylelint-config-recommended-scss": "13.1.0",
        "stylelint-config-recommended-vue": "1.6.0",
        "stylelint-config-standard": "34.0.0",
        "stylelint-config-standard-scss": "11.1.0",
        "stylelint-prettier": "4.1.0",
```

**Comment by Mouvedia** [comment]
Did you check [`@stylistic/stylelint-plugin`](https://www.npmjs.com/package/@stylistic/stylelint-plugin)?
Both rules are listed on [this page](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/docs/user-guide/rules.md).
